guides/common/modules/proc_accepting-a-self-signed-git-certificate.adoc

```diff
@@ -0,0 +1,12 @@
+[id="Accepting_a_Self-signed_Git_Certificate_{context}"]
+= Accepting a Self-signed Git Certificate
+
+If you are using a self-signed certificate authentication on your Git server, you must validate the certificate.
+
+.Procedure
+* Validate the certificate:
++
+[options="nowrap" subs="+quotes,verbatim,attributes"]
+----
+# sudo -u foreman git config --global http.sslCAPath _Path_To_My_Certificate_
```

Comment:
I'd say this step is roughly the HTTPS equivalent of accepting the SSH host key, which you have `in 13.2.1. Synchronizing Templates with an Existing Repository`.

Would it make sense to have 2 sub-procedures in `13.2.1. Synchronizing Templates with an Existing Repository`. One to set up SSH, one to set up HTTPS?

Reply:
Instead of splitting into two procedures, since there would be some common steps, I added validating the certificate as the last step. Please verify.